ide/rust/Syncer.kt

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
`Syncer` is run from a Bazel project that uses Rust. It identifies all the Bazel repositories that have Rust targets that are depended on by the current project, then runs the `RepoCargoManifestGenerator` in each one (plus the current project).

The end result is that the current project and all of its Bazel Rust dependencies will contain `Cargo.toml` files, allowing Cargo to build the project.